apps/web/src/components/Feed/Socialize/AdmireLine.tsx

```diff
@@ -47,37 +58,70 @@ export function AdmireLine({ admireRef, eventRef, queryRef, totalAdmires }: Comm
             }
           }
         }
+        ...useAdmireModalQueryFragment
+        ...isFeatureEnabledFragment
       }
     `,
     queryRef
   );
 
-  const event = useFragment(
-    graphql`
-      fragment AdmireLineEventFragment on FeedEvent {
-        dbid
-        ...NoteModalOpenerTextFragment
+  const isPfpEnabled = isFeatureEnabled(FeatureFlag.PFP, query);
```

Comment:
maybe worth it to `useMemo`?

Reply:
@kaitoo1 Im considering too but other places not wrapped with `useMemo` 🤔